wadm/lib/wadm/deployments/deployment_monitor.ex

```diff
@@ -84,20 +84,25 @@ defmodule Wadm.Deployments.DeploymentMonitor do
     # Make sure that there's a lattice supervisor running
     {:ok, _pid} = Wadm.LatticeSupervisor.start_lattice_supervisor(state.lattice_id)
 
-    Process.send_after(self(), :host_ping, 5_000)
+    Process.send_after(self(), :host_ping, 1_000)
     {:noreply, state}
   end
 
   @impl true
   def handle_info(:host_ping, state) do
-    if Gnat.pub(
+    # We pub instead of requesting here as we don't actually need to use the host
+    # inventories here, we create lattice state based on the heartbeats that are published
+    if Wadm.Nats.safe_pub(
          String.to_atom(state.lattice_id),
          "wasmbus.ctl.#{state.lattice_id}.ping.hosts",
          ""
        ) != :ok do
-      Logger.warn("Could not initiate host ping, first reconcilation pass may fail")
+      Logger.warning(
```

Comment:
I just realized there's both `warn` and `warning` macros :eye-twitch:

Apparently `warn` is deprecated? Good to know, but sad the more verbose option is preferred...